posthog/views.py

```diff
@@ -87,6 +88,7 @@ def preflight_check(request: HttpRequest) -> JsonResponse:
         "db": is_postgres_alive(),
         "initiated": User.objects.exists() if not settings.E2E_TESTING else False,  # Enables E2E testing of signup flow
         "cloud": settings.MULTI_TENANCY,
+        "realm": get_instance_realm(),
```

Comment:
Should we get rid of the `cloud` property? I'm worried about having more than one source of truth for something. Will cause headaches down the line.